protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2834,6 +2844,23 @@ message NetworkHistoryBootstrapPeersResponse {
     repeated string bootstrap_peers = 1;
 }
 
+// Request to export network history data in CSV format
+message ExportNetworkHistoryRequest {
+    // Block to begin exporting from. Must be the first block a history segment,
+    // which by default are 1000 blocks each - so 1, 1001 etc. are probably valid values.
```

Comment:
I don't think we can say `are probably valid values` in our docs :p

"Block to begin exporting from. Must be the first block a history segment" is probably all we need. We're already saying how to get a list of history segment in the docstring for `ExportNetworkHistory` we don't need it here aswell.